pkgs/development/python-modules/spsdk/default.nix

```diff
@@ -103,16 +88,23 @@ buildPythonPackage rec {
   ];
 
   nativeCheckInputs = [
-    pyftdi
+    ipykernel
+    pytest-notebook
     pytestCheckHook
     voluptuous
   ];
 
+  disabledTests = [
+    "test_nxpcrypto_create_signature_algorithm"
```

Comment:
also this should get a short comment

Reply:
We don't need to figure out why but we should at least comment how they failed, otherwise we can't determine in the future, if things got fixed or not.

Reply:
Can't you just run the test and see if it still fails?